openlibrary/plugins/openlibrary/js/compact-title/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Defines functions related to the compact title component.
+ * @module compact-title/index
+ */
+
+/**
+ * True if compact title component is visible on screen.
+ * @type {boolean}
+ */
+let isTitleVisible = false
```

Comment:
Not a blocker, but for future reference this state would be _much_ better as a class. This would be really awkward to unit test as is, trying to stub the actual module fields/methods.